HEVC stream reader - Out of Bounds comparison

Looking through the compilation build logs for another issue I saw this:
```
/Users/runner/runners/2.169.1/work/tsMuxer/tsMuxer/tsMuxer/hevcStreamReader.cpp:319:41: warning: result of comparison of constant 3981312000 with expression of type 'int' is always true [-Wtautological-constant-out-of-range-compare]
    else if (width <= 7680 && pixelRate <= 3981312000)
```
Looks to be the last comparison of the if/else tree to set DV `level`. Seems like `pixelrate` needs to be 64 bit?

--- a/tsMuxer/hevcStreamReader.cpp
+++ b/tsMuxer/hevcStreamReader.cpp
@@ -233,10 +233,13 @@ int HEVCStreamReader::setDoViDescriptor(uint8_t* dstBuff)
         m_hdr->isDVEL = true;
 
     int width = getStreamWidth();
-    int pixelRate = width * getStreamHeight() * getFPS();
+    uint32_t pixelRate = width * getStreamHeight() * getFPS();
 
     if (!isDVBL && V3_flags & FOUR_K)
+    {
+        width *= 2;
         pixelRate *= 4;
+    }
 
     // cf. "http://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-profiles-levels.pdf"
     int profile;

